File: src/main/java/com/aethercraft/moba/model/hero/Archer.java
package com.aethercraft.moba.model.hero;

public class Archer extends Hero {

	public Archer() {
		super("Archer", 17, 22, 17, new Ability[]{new Volley(), null, null, null});
	}

	public static class Volley extends Ability {
		public Volley() {
			super("Volley", Action.TargetPosition, Type.EnemyUnits, 100, 100, 10, new int[]{100,75,50,25}, new int[]{100,75,50,25}, false);
		}
	}
}

File: src/main/scala/com/aethercraft/moba/model/hero/Archer.scala
package com.aethercraft.moba.model.hero

object Volley extends Ability("Volley", AbilityEnum.Action.TargetPosition, AbilityEnum.Type.EnemyUnits, 100, 100, 10, Array(100,75,50,25), Array(100,75,50,25), false)

object Archer extends Hero("Archer", 17, 22, 17, Array[Ability](Volley, null, null, null))